programmers/kim.js

```diff
@@ -0,0 +1,15 @@
+function solution(seoul){
+    var location = seoul.indexOf("Kim");
+    var location2 = seoul.lastIndexOf("Kim");
```

Comment:
var, let, const의 차이를 아셔야합니다.
indexOf도 쓰시고 굳입니다.
```suggestion
    let location = seoul.indexOf("Kim");
    let location2 = seoul.lastIndexOf("Kim");
```